java/runtime/src/main/java/io/ray/runtime/object/ObjectStore.java

```diff
@@ -31,6 +31,15 @@ public ObjectStore(WorkerContext workerContext) {
    */
   public abstract ObjectId putRaw(NativeRayObject obj);
 
+  /**
+   * Put a raw object into object store, and assign ownership to actor.
+   *
+   * @param obj The ray object.
+   * @param address The address of the actor to assign ownership
+   * @return Generated ID of the object.
+   */
+  public abstract ObjectId putRaw(NativeRayObject obj, byte[] address);
```

Comment:
I leave this overloaded function untouched, because otherwise it makes the code hard to read. By just passing null, java couldn't decide which method to call. So we need to either reverse the args, or claim a dummy variable just to match the type.